eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Common.php

```diff
@@ -168,6 +168,7 @@ public function mapConfig(array &$scopeSettings, $currentScope, ContextualizerIn
             $contextualizer->setContextualParameter('binary_dir', $currentScope, $scopeSettings['binary_dir']);
         }
         if (isset($scopeSettings['api_keys']['google_maps'])) {
+            $contextualizer->setContextualParameter('api_keys', $currentScope, ['google_maps' => $scopeSettings['api_keys']['google_maps']]);
             $contextualizer->setContextualParameter('api_keys.google_maps', $currentScope, $scopeSettings['api_keys']['google_maps']);
```

Comment:
@bdunogier I'm a bit unsure why I need both to make sure both covered test cases work.